opentech/apply/activity/messaging.py

```diff
@@ -397,6 +397,9 @@ class SlackAdapter(AdapterBase):
         MESSAGES.UPDATE_PAYMENT_REQUEST_STATUS: '{user} has updated status for payment request <{link}|{source.title}>.',
         MESSAGES.DELETE_PAYMENT_REQUEST: '{user} has deleted payment request from <{link}|{source.title}>.',
         MESSAGES.UPDATE_PAYMENT_REQUEST: '{user} has updated payment request for <{link}|{source.title}>.',
+        MESSAGES.PROJECT_MOVED_TO_CLOSING: '{user} has moved project to closing <{link}|{source.title}>.',
+        MESSAGES.PROJECT_MOVED_TO_CLOSED: '{user} has moved project to closed <{link}|{source.title}>.',
+        MESSAGES.PROJECT_MOVED_TO_IN_PROGRESS: '{user} has moved project to in progress <{link}|{source.title}>.',
```

Comment:
Can we have activities for these as well please.

`Moved to <state>`

Reply:
Fixed and rebased.